README.md

```diff
@@ -4,6 +4,7 @@ JSON support for Sublime's LSP plugin.
 
 Uses [VSCode JSON Language Server](https://github.com/Microsoft/vscode/tree/master/extensions/json-language-features/server) to provide validation, formatting and other features for JSON files. See linked repository for more information.
 
+* Make sure you have Node.js installed and `node` is in your `$PATH`. The language server subprocess is a Node.js app.
```

Comment:
We could sneak in a link to https://lsp.readthedocs.io/en/latest/troubleshooting/#2-lsp-cannot-find-my-language-server-no-such-file-or-directory-xyz in there.